.gitignore

```diff
@@ -1,7 +1,5 @@
-/dist
```

Comment:
dist directory does not exist in our library so removing it.

Reply:
I think that folder is where the distribution files are usually placed. For example the min in the map files might be generated in when running npm build dist or some similar command. Also, what happened to generating a file for the latest version?